app/src/main/java/org/novinger/jason/demoreader/datamodels/LeadArt.java

```diff
@@ -0,0 +1,28 @@
+package org.novinger.jason.demoreader.datamodels;
+
+/**
+ * Created by jason on 5/1/16.
+ */
+public class LeadArt {
```

Comment:
so this is fine, and is probably how i'd do it too, but fwiw you could probably just use a `HashMap` instead of this class